crates/swc_plugin_runner/src/host_environment.rs

```diff
@@ -11,12 +9,10 @@ use wasmer::Memory;
 ///
 /// ref: https://docs.wasmer.io/integrations/examples/host-functions#declaring-the-data
 #[derive(Clone)]
-pub struct BaseHostEnvironment {
-    pub memory: Option<Memory>,
-}
+pub struct BaseHostEnvironment {}
```

Comment:
Do we still need this type?

Reply:
This isn't needed, but I'm keep it in case I might need to put something in future.